recipes/chemfiles-python/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set version = "0.6.0" %}
+{% set sha256 = "913a9ca911287e342918458619c29dfd094f4b25f7d6866048b9968fc63be207" %}
+
+package:
+  name: chemfiles-python
+  version: {{ version }}
+
+about:
+  home: http://chemfiles.github.io
+  license: MPL-v2.0
+  summary: Python binding to chemfiles, a modern library for chemistry file reading and writing
+  doc_url: http://chemfiles.github.io/chemfiles.py/latest/
+  dev_url: https://github.com/chemfiles/chemfiles.py
+
+build:
+  script:  # [not win]
+      - export MACOSX_DEPLOYMENT_TARGET=""  # [osx]
+      - python setup.py install # [not win]
+  script:  # [win]
+      - "%PYTHON%" setup.py install # [win]
+      - if errorlevel 1 exit 1  # [win]
+source:
+  fn: chemfiles.py-{{ version }}.tar.gz
+  url: https://github.com/chemfiles/chemfiles.py/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+  patches:
+      - find_library.patch
+
+test:
+  commands:
+    - cd $SRC_DIR/tests
+    - python -m unittest discover
+
+requirements:
+  build:
+    - python
+    - enum34  # [py < 34]
+    - numpy
+    - chemfiles-lib ==0.6.0
```

Comment:
Are these all required at build time? Guessing not. Please drop everything other than `python`. Also please add `setuptools` as that seems to be required for the build.